rest/route/service.go

```diff
@@ -165,6 +165,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/user/settings").Version(2).Post().Wrap(checkUser).RouteHandler(makeSetUserConfig(sc))
 	app.AddRoute("/users/{user_id}/hosts").Version(2).Get().Wrap(checkUser).RouteHandler(makeFetchHosts(sc))
 	app.AddRoute("/users/{user_id}/patches").Version(2).Get().Wrap(checkUser).RouteHandler(makeUserPatchHandler(sc))
+	app.AddRoute("/users/{user_id}/clear_hosts").Version(2).Post().Wrap(checkUser, editRoles).RouteHandler(makeClearHostsByUser(sc, env))
```

Comment:
why does this use the `editRoles` middleware?

Reply:
Maybe @john-m-liu will have a better suggestion: I chose this because I figured the users who can edit roles should also be the same people who can clear_hosts. Should I instead create a new role? Who would I give this to for this to be usable by MANA?

Reply:
This is the right role to be usable by mana. It does feel a bit weird, but if you think of this as a route to offboard a user, then I think makes sense. And maybe we should actually name it to offboarding a user, so that we can add to it in the future without making mana change anything

Reply:
Oh makes sense to me.